specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2022-04-01/workbooks_API.json

```diff
@@ -650,9 +656,7 @@
         },
         "innerError": {
```

Comment:
>innerError [](http://example.com/codeflow?start=9&length=10)

expected workbook error to be:

    "body": {
      "error": {
        "code": "aaaaaaaaaa",
        "message": "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
        "innererror": {
          "trace": [
            "aaaaaaaaaaaaaaaaaaaaaaaaaaaa"
          ]
        }
      }
    }